packages/data-connect/test/dataconnect/dataconnect.yaml

```diff
@@ -1,11 +1,12 @@
 specVersion: "v1beta"
-serviceId: "dataconnect"
+serviceId: "fdc-service"
 location: "us-west2"
 schema:
   source: "./schema"
   datasource:
     postgresql:
-      database: "dataconnect-test"
+      database: "tests"
       cloudSql:
-        instanceId: "local"
+        instanceId: "prod"
+      # schemaValidation: "COMPATIBLE"
```

Comment:
nit: remove commented code, or expand the comment to explain why it's commented and why one may want to uncomment it.

Reply:
This is what's generated by the CLI, so I'm just going to keep it as-is